code/solr/Solr.php

```diff
@@ -227,6 +227,8 @@ public function run($request)
     {
         parent::run($request);
 
+        $this->extend('updateBeforeSolrConfigureTask', $request);
```

Comment:
`parent::run()` runs before this... Should this go first?

Reply:
I put it after the parent::run() call as I wasn't sure where it was in the code that the translatable filters were getting reset in my particular scenario and wanted to make sure that I was able to toggle that filter just before the actual index action took place.